packages/react/src/primitives/Pagination/usePaginationItems.tsx

```diff
@@ -66,12 +71,13 @@ export const usePaginationItems = (
             type="page"
             page={item as number}
             currentPage={currentPage}
+            currentPageLabel={currentPageLabel}
             onClick={onChange}
-            ariaLabel={`Go to page ${item}`}
+            ariaLabel={`${pageLabel} ${item}`}
```

Comment:
I see that we're really just porting these changes over so as to not introduce a breaking change, so this is also out of scope, but want to point this out so we might reference it for future discussions: When we do a review of our i18n capabilities, we should consider how we would support interpolation in our string translations. Example, as is this works for "Go to page 31" or "translatedText {s}" as the supplied string, but for Arabic or Japanese or some other languages the supplied string might look like: 
"{s} translatedText" or even "translatedText {s} translatedText". No suggested changes now, just putting this down somewhere so we can point to it.

Reply:
Oh yes, definitely a good call out here. We have some accessibility debt here. Thanks for the insights! Maybe I should add a TODO there to remind us as well